lib/insert-link.js

```diff
@@ -54,8 +54,11 @@ function getCaptureGroupValue(match, captureGroup) {
   return captureGroup.replace(new RegExp(`\\$${index}`, 'g'), match[index]);
 }
 
-function buildDataAttr(data, match) {
-  const dataAttr = {};
+function buildDataAttr(_pluginName, data, match) {
+  const dataAttr = {
+    _pluginName,
```

Comment:
EDIT: See this comment first: https://github.com/OctoLinker/browser-extension/pull/274/files#r99663118

If you agree with my comment at https://github.com/OctoLinker/browser-extension/pull/274/files#r99657441, we should also change `_pluginName` to `pluginName` on this  line (and in the function definition).